server/src/lib/api/api.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+import express from 'express';
+import modelFinder from '../middleware/modelFinder';
+
+const router = express.Router();
+router.param('model', modelFinder);
+
+router.post('/api/v1/:model', (req, res, next) => {
+  let document = new req.model(req.body);
+
+  document.save()
+    .then(data => { res.status(200).json(data); })
+    .catch(next);
+});
+
+router.get('/api/v1/:model/:id', (req, res, next) => {
+
+  req.model.findById(req.params.id)

```

Comment:
Change to "findOne" or just "find" since "findById" doesn't seem to work after testing it. Looks like there have been prior issues with mongoose related to "findById" not populating properly.